src/core/manager.c

```diff
@@ -1093,6 +1093,9 @@ static int manager_setup_notify(Manager *m) {
         }
 
         if (!m->notify_event_source) {
+                const usec_t NOTIFY_RATE_LIMIT_INTERVAL = 1 * USEC_PER_SEC;
+                const unsigned NOTIFY_RATE_LIMIT_BURST = 10;
```

Comment:
We don't do const variables like this. Please define these as macros at the top of manager.c

Reply:
Fixed